### PR TITLE
Integrity check function for simple_io

### PR DESCRIPTION
**Pull request by shylesh**
Signed-off-by: Shylesh Kumar Mohan <shmohan@redhat.com>

### CODE REVIEW

**Review by petr-balogh — DISMISSED** [review]

File: ocs_ci/ocs/dr/dr_workload.py [hunk 1]
```diff
@@ -138,3 +142,82 @@ def delete_workload(self):
 
         config.switch_to_cluster_by_name(primary_cluster_name)
         dr_helpers.wait_for_workload_resource_deletion(self.workload_namespace)
+
+    def data_integrity_check(self, workload_namespace=None, workload_pattern=None):
```
Comment (sidhant-agrawal) [inline ocs_ci/ocs/dr/dr_workload.py:146]: If this method is common for all workloads (busybox and mysql), maybe we can move it to base class.

**Comment by keesturam** [comment]
@shylesh could you please rebase and resend this PR for review.

**Comment by shylesh** [comment]
Rebased on latest master

File: ocs_ci/ocs/dr/dr_workload.py [hunk 2]
```diff
@@ -253,3 +257,84 @@ def delete_workload(self, force=False):
...
+        If both namespace and pattern are null then we will fetch a list of all the pods from all the workload
+        namespaces and run integrity check on all of them.
+
+        Args:
+            workload_namespace (str): Namespace in which workload has been deployed
+            workload_pattern (list): workload patterns like ["workloadpattern=simple_io", "workloadpattern=fio"]
+                                    etc which will be label
+
+        Returns:
+            3 tuple: (constants.DR_WORKLOAD_PASS/FAIL, output/exception, pod)
```
Comment (petr-balogh) [inline ocs_ci/ocs/dr/dr_workload.py:283]:
```suggestion
            tuple: (constants.DR_WORKLOAD_PASS/FAIL, output/exception, pod)
```

File: ocs_ci/ocs/dr/dr_workload.py [hunk 3]
```diff
@@ -253,3 +257,84 @@ def delete_workload(self, force=False):
...
+            )
+            for namespace in wl_namespaces:
+                pod_list.extend(
+                    pod.get_all_pods(
+                        namespace=namespace, selector=constants.DR_WORKLOAD_POD_LABEL_ID
+                    )
+                )
+
+        future_to_pods = {}
+        # List of 3 tuple (retval(0 for pass , 1 for fail) flag, exception/output, pod)
```
Comment (petr-balogh) [inline ocs_ci/ocs/dr/dr_workload.py:317]: Just one tuple with three items

**Pull request closed without merging**